internal/constants/constants.go

```diff
@@ -6,6 +6,9 @@ const ProjectID = "erouska-key-server-dev"
 //FirebaseURL URL to our firebase DB.
 const FirebaseURL = "https://" + ProjectID + ".firebaseio.com/"
 
+//KeyServerURL URL to Key server (w/o path)
+const KeyServerURL = "https://exposure-i5jzq6zlxq-ew.a.run.app/"
```

Comment:
this shouldn't be hardcoded and ideally not visible publicly... make it configurable via ENV variable please